modules/cats/src/test/scala/pureconfig/module/cats/CatsSuite.scala

```diff
@@ -0,0 +1,23 @@
+package pureconfig.module.cats
+
+import cats.data.{ NonEmptyList, NonEmptyVector }
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+import pureconfig.syntax._
+
+class CatsSuite extends FlatSpec with Matchers with EitherValues {
+
+  case class Numbers(numbers: NonEmptyList[Int])
+
+  it should "be able to read a config with a NonEmptyList" in {
+    val config = ConfigFactory.parseString(s"""{ numbers: [1,2,3] }""")
+    config.to[Numbers].right.value shouldEqual Numbers(NonEmptyList(1, List(2, 3)))
```

Comment:
You're right, I didn't know about `EitherValues` and I assumed `.value` was an alias for `.get` on a `RightProjection`, and not an extension method provided by ScalaTest.

Personally, I still prefer the less magical equality comparison between the `Either` and a `Right(expected)`, but I have no issue with the way it was :)